packages/shared/forks/ReactFeatureFlags.www-dynamic.js

```diff
@@ -60,4 +60,4 @@ export const enableSyncDefaultUpdates = __VARIANT__;
 export const allowConcurrentByDefault = true;
 export const enablePersistentOffscreenHostContainer = false;
 
-export const consoleManagedByDevToolsDuringStrictMode = true;
+export const consoleManagedByDevToolsDuringStrictMode = __VARIANT__;
```

Comment:
The instructions in this file tell me to use the VARIANT configuration instead of hardcoding a value so I assume that's what I should do

Reply:
If I understand correctly those were originally TODOs that failed when this file was introduced. At the time the assumption was that we’d convert them all, and that any newly added ones should be added above (and use VARIANT). Maybe we should make that comment clearer.